docs/spark-connector/spark-authentication-with-gravitino.md

```diff
@@ -0,0 +1,47 @@
+---
+title: "Spark authentication with Gravitino server"
+slug: /spark-connector/spark-authentication
+keyword: spark connector authentication oauth2 kerberos
+license: "This software is licensed under the Apache License version 2."
+---
+
+## Overview
+
+Spark connector supports `simple` `oauth2` and `kerberos` authentication when accessing Gravitino server.
+
+| Property                     | Type   | Default Value | Description                                                                                                         | Required | Since Version |
+|------------------------------|--------|---------------|---------------------------------------------------------------------------------------------------------------------|----------|---------------|
+| spark.sql.gravitino.authType | string | `simple`      | The authentication mechanisms when communicating with Gravitino server, supports `simple`, `oauth2` and `kerberos`. | No       | 0.7.0         |
+
+## Simple mode
+
+In the simple mode, you could specify the username by configuration.
+
+| Property                            | Type   | Default Value | Description                           | Required | Since Version |
+|-------------------------------------|--------|---------------|---------------------------------------|----------|---------------|
+| spark.sql.gravitino.simple.userName | string | None          | The user name to access the Gravitino | No       | 0.7.0         |
+
+If you doesn't set `spark.sql.gravitino.simple.userName` explicitly. It will use the value of environment variable `GRAVITINO_USER` as the user. If the environment variable `GRAVITINO_USER` isn't set, the client uses the user logging in the machine.
```

Comment:
So the default value of `spark.sql.gravitino.simple.userName` is not `None`?

Reply:
updated